modules/backup-deployment-config-operator.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/backing-up-acs.adoc
+:_content-type: PROCEDURE
+[id="backup-deployment-config-operator_{context}"]
+= Backup deployment using the {product-title-short} Operator
+
+[role="_abstract"]
+When you use the {product-title-short} Operator to instal {product-title-short}, {ocp} stores all the custom configuration for your Central deployment within the Central custom resource.
+You can backup the Central custom resource, `central-tls` and the administrator password. `central-tls` includes the certificates for authenticating with Secured clusters and signing API tokens.
```

Comment:
I think another comma is needed in the list? Altho I wasn't sure if `central-tls` is the `Secret` name or the "backup file" as it's referred to elsewhere. Describing a bit more what `central-tls` is here might help. E.g.:

```suggestion
You can backup the Central custom resource, the `central-tls` secret, and the administrator password.  The `central-tls` secret includes the certificates for authenticating with Secured clusters and signing API tokens.
```

Or `s/secret/backup file/` if that's more accurate at this stage.